automation/test.sh

```diff
@@ -443,6 +445,8 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     label_filter='(VGPU)'
   elif [[ $TARGET =~ sev.* ]]; then
     label_filter='(SEV)'
+  elif [[ $TARGET =~ secure-execution ]]; then
+    label_filter='(secure-execution)'
   elif [[ $TARGET =~ sig-compute-realtime ]]; then
     label_filter='(sig-compute-realtime) && !(SEV, SEVES)'
```

Comment:
the secure-execution should be filtered out of the regular compute lanes similar to SEV